src/Components/SearchResults.js

```diff
@@ -1,14 +1,25 @@
-import React from "react";
+import React, { useState } from "react";
 import Columns from "../Components/Table/Columns";
 import Rows from "../Components/Table/Rows";
 
 const SearchResults = props => {
-  const objDataFakeBooking = props.results[0];
+  const [selectedRow, setSelectedRow] = useState(null);
```

Comment:
This is a really good attempt, but it looks like if you click on any row, then all of them become highlighted. The exercise asks you to make each row *independent*, meaning that multiple rows can be highlighted at once.

You're really close, can you work out how to do that?

Reply:
`  <tr
      key={props.data.id}
      className={selectedRow ? "select" : ""}
      onClick={() => {
        setSelectedRow(!selectedRow);
      }}
    >`